examples/k8s.yaml

```diff
@@ -78,27 +86,32 @@ data:
             max_connection_age-grace: 30s
       jaeger: {}
       zipkin: {}
-# Can only use one exporter
-#    exporters:
-#      jaeger:
-#        collector_endpoint: "http://jaeger.default:14268/api/traces"
-#      zipkin: {}
-    queued-exporters:
-      omnition:
-        batching:
-          enable: true
-          timeout: 5s
-          send_batch_size: 1024
+    processors:
+      queued_retry:
         num_workers: 16
         queue_size: 10000
         retry_on_failure: true
-        opencensus:
-          compression: "gzip"
-          endpoint: "ingest.omnition.io:443"
-          headers: { "x-omnition-api-key":"00000000-0000-0000-0000-000000000001" } # TODO: Update me
-          num_workers: 8
-          reconnection_delay: 2s
-          secure: true
+      batch:
+        send_batch_size: 1024
+        timeout: 5s
+    extensions:
+      health_check: {}
+    exporters:
+      opencensus:
+        compression: "gzip"
+        endpoint: "someendpoint.provider.com:443" # Replace with a real endpoint
+        num_workers: 8
+        reconnection_delay: 2s
+        secure: true
+      zipkin: {}
+      jaeger: {}
```

Comment:
wasn't jaeger split into the `jaeger_grpc` and `jaeger_thrift_http` exporters?

Reply:
Good catch - didn't realise that happened in the v0.2.0 release updating that now